CommonTools/TriggerUtils/plugins/CandidateTriggerObjectProducer.cc

```diff
@@ -88,8 +88,8 @@ void CandidateTriggerObjectProducer::produce(edm::Event& iEvent, const edm::Even
     //matching with regexp filter name. More than 1 matching filter is allowed
     if (TString(*iHLT).Contains(TRegexp(TString(triggerName_)))) {
       triggerInMenu[*iHLT] = true;
-      const std::pair<int, int> prescales(hltPrescaleProvider_.prescaleValues(iEvent, iSetup, *iHLT));
-      if (prescales.first * prescales.second == 1)
+      auto const prescales = hltPrescaleProvider_.prescaleValues<double>(iEvent, iSetup, *iHLT);
+      if (prescales.first == 1 and prescales.second == 1)
```

Comment:
See https://github.com/cms-sw/cmssw/pull/39220#discussion_r975393631. Here, I changed the check to "HLT==1 and L1T==1", instead of doing a floating-point operation ("HLT * L1T").